lib/coins/ada.js

```diff
@@ -0,0 +1,41 @@
+const b58 = require('b58')
```

Comment:
This package has 8 downloads in a week so I am quite afraid of using it. However, I believe there should be something like that already available as other cryptocurrencies should need it. Could you please check current dependencies or how other currencies do this?

Reply:
It seems you can use already install "base-x" package.

Reply:
There is also a `bs58check` package, but that does BTC (and other similar currencies) specific CRC checking (Cardano have different algorithm), so from this point of view `base-x` looks good.